cardinal/system/system_inventory_system.go

```diff
@@ -59,7 +59,7 @@ func Inventory(playerID uint32, ts *TokeniserSystem, world cardinal.WorldContext
 
 func Take(tokens []string, playerID uint32, roomID uint32, ts *TokeniserSystem, world cardinal.WorldContext) (string, uint8) {
 	var tok_err uint8
-	var hasBeenPickedUp bool
+	var PickedUp bool
```

Comment:
minor but I think this should be "pickedUp" the capitalisation is not idiomatic go

Reply:
done